core/fslock/fslock_error.hpp

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef FILECOIN_CORE_FSLOCK_FSLOCK_ERROR_HPP
+#define FILECOIN_CORE_FSLOCK_FSLOCK_ERROR_HPP
+
+#include "common/outcome.hpp"
+
+namespace fc::fslock {
+
+  /**
+   * @brief FSLock returns these types of errors
+   */
+  enum class FSLockError {
+    FILE_LOCKED = 1,
+    IS_DIRECTORY = 2,
```

Comment:
you don't have to assign numbers to all entries, only to the first one

Reply:
Fixed